tests/src/io.jsonwebtoken/jjwt-gson/0.11.5/src/test/java/io_jsonwebtoken/jjwt_gson/Jjwt_gsonTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package io_jsonwebtoken.jjwt_gson;
+
+import io.jsonwebtoken.CompressionCodecs;
+import io.jsonwebtoken.JwtParserBuilder;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.io.Encoders;
+import io.jsonwebtoken.security.Keys;
+import org.junit.jupiter.api.Test;
+
+import javax.crypto.SecretKey;
+import java.util.Date;
+import java.util.UUID;
+import java.util.stream.Stream;
+
+/**
+ * @author linghengqian
+ */
+class Jjwt_gsonTest {
+    @Test
+    void testSignedJWTs() {
+        SecretKey firstKey = Keys.secretKeyFor(SignatureAlgorithm.HS256);
+        assert Jwts.parserBuilder().setSigningKey(firstKey).build().parseClaimsJws(
```

Comment:
Please use `assertJ` like the other tests do instead of plain java asserts.

Reply:
- I finished.

Reply:
- I left out part of the unit test unmodified, which I'll deal with later in this PR.

Reply:
- done.